rpm/stackdriver-agent.spec

```diff
@@ -42,10 +58,28 @@
 %define bundle_yajl 1
 %endif
 
+%if 0%{?suse_version} > 0
+%define dep_filter 0
+%define bundle_curl 0
```

Comment:
A lot of the conditions seem to be around `bundle_curl`. Is this because SUSE does not support `libcurl`?

Reply:
It's because SUSE doesn't require us to provide our own `libcurl`.